docs/data/base/getting-started/customization/customization.md

```diff
@@ -0,0 +1,83 @@
+# Customizing MUI Base components
+
+<p class="description">There are several ways to customize MUI Base components, from applying custom CSS rules to building fully custom components using hooks.</p>
+
+With MUI Base, you have the freedom to decide how much you want to customize a component's structure and style.
+
+This document reviews the three levels of customization that are available: applying custom CSS rules, overriding default subcomponent slots, and using hooks to build fully custom components.
+
+## Applying custom CSS rules
+
+If you're happy with the default structure of a component's rendered HTML, you can apply custom styles to the component's classes.
+
+Each component has its own set of classes.
+Some classes are **static**, which is to say that they are always present on the component.
+Others are applied **conditionally**—like `Mui-disabled`, for example, which is only present when a component is disabled.
+
+Each component's API documentation lists all classes that the component uses.
+Additionally, you can import a `[componentName]Classes` object that describes all the classes a given component uses, as the following demo illustrates:
+
+{{"demo": "StylingCustomCss.js"}}
+
+## Overriding subcomponent slots
+
+If you want to make changes to a component's rendered HTML structure, you can override the default subcomponents ("slots") using the `components` and/or `component` prop—see ["Shared props" on the Base Usage page](/base/getting-started/usage/#shared-props) for more details.
+
+The following demo uses [SwitchUnstyled](/base/react-switch/) to show how to create a styled component by applying styles to three of its subcomponent slots: `Root`, `Thumb`, and `Input`.
+
+Note that although this demo uses [MUI System](/system/styled/) as a styling solution, you are free to choose any alternative.
+
+{{"demo": "StylingSlots.js"}}
+
+The components you pass in the `components` prop receive the `ownerState` prop from the top-level component (the "host").
```

Comment:
We settled on calling the top-level component "owner" after all (hence "ownerState"). "Host" is likely a leftover from the past.

```suggestion
The components you pass in the `components` prop receive the `ownerState` prop from the top-level component (the "owner").
```

While "host" may look better at first glance, it can be confused with React concept of "host components", which are entirely unrelated.

Reply:
Ah good to know, thanks! I'll update the language here and look out for it in other pages that mention this.